pinot-core/src/main/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -96,9 +97,9 @@
   private final int _numKeyColumns;
 
   // default message metadata
-  private static final StreamMessageMetadata _defaultMetadata = new StreamMessageMetadata();
-  private long _lastIndexedTimestamp = Long.MIN_VALUE;
-  private long _latestIngestionTimestamp = Long.MIN_VALUE;
+  private static final StreamMessageMetadata _defaultMetadata = new StreamMessageMetadata(System.currentTimeMillis());
+  private volatile long _lastIndexedTimestamp = Long.MIN_VALUE;
+  private volatile long _latestIngestionTimestamp = Long.MIN_VALUE;
```

Comment:
(Optional) **_lastIndexedTimeMs** and **_latestIngestionTimeMs** to be more clear?